docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -222,6 +252,15 @@ updates to its `.spec.template` will not be propagated to its Pods.
 In most cases you will not need to use a partition, but they are useful if you want to stage an
 update, roll out a canary, or perform a phased roll out.
 
+## Selectors and Labels
```

Comment:
Remove this section ... StatefulSet selector was never mutable.

Reply:
Sorry - I am aware that there is no change to statefulset strategy in my previous PR, but didn't know that statefulset selector was never mutable.